gui/browserv7/src/RBrowser.cxx

```diff
@@ -291,8 +291,8 @@ RBrowser::RBrowser(bool use_rcanvas)
    fWebWindow->SetDefaultPage("file:rootui5sys/browser/browser.html");
 
    // this is call-back, invoked when message received via websocket
-   fWebWindow->SetCallBacks([this](unsigned connid) { fConnId = connid; SendInitMsg(connid); },
-                            [this](unsigned connid, const std::string &arg) { ProcessMsg(connid, arg); });
+   fWebWindow->SetCallBacks([this](unsigned connid) { if (!fConnId) { fConnId = connid; SendInitMsg(connid); } else fConnId = 0xffffff; },
+                            [this](unsigned connid, const std::string &arg) { if ((connid == fConnId) && (fConnId != 0xffffff)) ProcessMsg(connid, arg); });
```

Comment:
This means I need to try on average 8388608 times until the server lets me in. I can apparently get 5k connections per second from civetweb, so that's 30 minutes. Is this good enough? This is just the average; if I try this for 1000 users I *will* find a connection that responds, way earlier (P=p vs P=1-(1-p)^1000).

Reply:
No. You need to have `0xfffffe` active connections to get `0xfffffff` as connection id. 
Normally it is limited by number of threads started in `civetweb` server.
Even than - any new connection will be rejected as long as `fConnId == 0xffffff`

Reply:
> No. You need to have 0xfffffe active connections to get 0xfffffff as connection id.

Oh but that's *terrible*! Then I basically *know* the connection ID as an attacker.

Reply:
RBrowser will accept connection only once. Point.

With any next connection id will be set as 0xffffff. With id == 0xffffff RBrowser ignores any data.